util/kubectl_test.go

```diff
@@ -0,0 +1,10 @@
+package util
+
+import "testing"
+
+func TestCheckKubectl(t *testing.T) {
+	result := CheckKubectl()
```

Comment:
Yes you can install packages in the circle.yml so that should work. I think that makes sense especially if you want to write more tests that need kubectl.

https://circleci.com/docs/1.0/installing-custom-software/